rust/migrations/20221128012310_add_payments.sql

```diff
@@ -0,0 +1,12 @@
+-- Add lightning payments table
+CREATE TABLE IF NOT EXISTS payments (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    payment_hash TEXT UNIQUE NOT NULL,
+    preimage TEXT,
+    secret TEXT,
+    htlc_status TEXT NOT NULL,
+    amount_msat INTEGER,
```

Comment:
❓ How can the amount be null?

Reply:
I guess it depends on what a `payment` is - an `invoice` can be created without an amount (and then the payer chooses the amount). 
For me a payment would identify something that has a concrete amount that is either incoming or outgoing. That said I think this should be `NOT NULL` I guess...?

Reply:
unfortunately, it can be null, according to the rust-lightning people ;/ I am keeping it as close as possible to the current model (which actually supports null).